### PR TITLE
Update to ScalaNative 0.3.1; Bump version number to 3.6.1

### LINKED ISSUE #146: Update to ScalaNative 0.3.1
ditto

### PR DESCRIPTION
**Pull request by frgomes**
Fix #146

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ import com.typesafe.sbt.pgp.PgpKeys._
 // shadow sbt-scalajs' crossProject and CrossType until Scala.js 1.0.0 is released
 import sbtcrossproject.{crossProject, CrossType}
 
-def v: String = "3.6.0"
+def v: String = "3.6.1"
```
Comment (eed3si9n) [inline build.sbt:6]: I can probably back publish against 3.6.0.

**Review by eed3si9n — CHANGES_REQUESTED** [review]

**Comment by eed3si9n** [comment]
https://oss.sonatype.org/content/repositories/public/com/github/scopt/scopt_native0.2_2.11/

**Comment by ShaneDelmore** [comment]
Thanks for the quick fix and publish @frgomes and @eed3si9n 👍

**Comment by eed3si9n** [comment]
I haven't published yet, I was thinking if @frgomes wants to fix the commit, I can merge just the project/site.sbt change. I can make the change locally too.

**Comment by eed3si9n** [comment]
https://github.com/scopt/scopt/commit/9f99f90be641dd0908b9cc5f329c0151bde27dbf
https://oss.sonatype.org/content/repositories/public/com/github/scopt/scopt_native0.3_2.11/3.6.0/

**Comment by ShaneDelmore** [comment]
Oops, I misread the repo link in my excitement.

**Pull request closed without merging**